src/Session.cpp

```diff
@@ -115,12 +115,18 @@ SResult SESSION::CSession::Initialize(std::string manifestUrl)
 
   m_adaptiveTree = PLAYLIST_FACTORY::CreateAdaptiveTree(manifestResp);
   if (!m_adaptiveTree)
-    return SResult::Error("Unable to determine type of manifest file.");
+  {
+    LOG::LogF(LOGERROR, "Unable to determine type of manifest file.");
+    return SResult::Error("A problem occurred in the stream initialization. See the log for details.");
```

Comment:
1) If you want to make the messages less technical, then pointing users to the log is not a good approach. Most of the users have no idea how to read the log, and many of them do not even know whereto find the log. For Kod we removed the "see log for details" part of many error message strings.

2) If this is for end users, these string should be localizable.

Reply:
done